strapi/strapi
# ML: Empty state is shown after deleting all assets from a page

Fixes #12648

### What does it do?

- After deleting all assets from a page that is not the first page, a user is routed to the previous page that still has assets

### Why is it needed?

- After deleting all assets from a page that is not the first page, a user stayed on  that page showing an empty page without assets even when the previous page has assets.

### How to test it?


1. From the menu navigate to the Media Library
2. Add enough assets to be displayed on 2 pages.
3. Navigate to the second page and delete all its assets
    - After successful deletion, you should be routed to page 1 that still has assets

#### Demo
https://user-images.githubusercontent.com/45232708/205258689-04da6210-440c-41dc-b714-ae8341dbdfa0.mov

### Related issue(s)/PR(s)
